src/alert-manager/deploy/alert-manager-configmap.yaml.template

```diff
@@ -32,8 +32,12 @@ data:
       repeat_interval: {{ cluster_cfg["alert-manager"]["repeat-interval"] }}
       group_by: [alertname, alertstate]
 
-      {% if 'routes' in cluster_cfg["alert-manager"]["customized-routes"] %}
       routes:
+      - receiver: pai-cordon-nodes
+        match:
+          alertname: NvidiaSmiEccError
```

Comment:
Seems only need to match double EccError here. Please only match the alert when label `type` is `double`